.github/workflows/homebrew-core-head.yml

```diff
@@ -11,6 +11,10 @@ jobs:
     name: Build Semgrep via Brew from `returntocorp/semgrep:develop`
     runs-on: ["self-hosted", "macOS", "X64"]
     steps:
+      - name: Uninstall semgrep
+        # This is sub-optimal - our workflows shouldn't have to conform to their environment.
+        # However, on the runner side, we can't hook into the workflow run to clean up after.
+        run: brew uninstall semgrep || true
```

Comment:
Perhaps we should also run this as a cleanup step at the end of the job. That way the assumption can be that on a nominal run it resets the environment back to start.

Reply:
I like that idea.